### PR TITLE
548 - CI - Add doc-changelog action

### PR DESCRIPTION
**Pull request by da1910**
Add the doc-changelog action to update the markdown file as per instructions at:

https://actions.docs.ansys.com/version/stable/migrations/docs-changelog-setup.html#docs-changelog-action-setup

### CODE REVIEW

**Comment by da1910** [comment]
CI is blocked by https://github.com/microsoft/linux-package-repositories/issues/130

File: pyproject.toml [hunk 1]
```diff
@@ -154,3 +154,38 @@ files = "src"
...
+
+[[tool.towncrier.type]]
+directory = "dependencies"
+name = "Dependencies"
+showcontent = true
+
+[[tool.towncrier.type]]
+directory = "miscellaneous"
+name = "Miscellaneous"
+showcontent = true
```
Comment (ludovicsteinbach) [inline pyproject.toml:191]: Shouldn't these match the labels that we use in the repo?

Comment (ludovicsteinbach) [reply pyproject.toml:191]: Oh right, there is a hardcoded mapping in the action https://github.com/ansys/actions/blob/7c46e54da0e3d73141e79f41a63a43a24cb855b8/doc-changelog/action.yml#L120

Comment (da1910) [reply pyproject.toml:191]: Hmm, these are the headings in the changlog, and they're the ones that the PyAnsys team recommends, they have a map you can see here:

https://github.com/ansys/actions/blob/7c46e54da0e3d73141e79f41a63a43a24cb855b8/doc-changelog/action.yml#L120-L125

**Review by ludovicsteinbach — APPROVED** [review]

**Pull request merged**